### ISSUE TITLE
Separate Youtube Shorts kiosk

### ISSUE THREAD

**Issue by moonstarsozluk** [TeamNewPipe/NewPipe#8518]
### Checklist

- [X] I made sure that there are *no existing issues* - [open](https://github.com/TeamNewPipe/NewPipe/issues) or [closed](https://github.com/TeamNewPipe/NewPipe/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I'm aware that this is a request for NewPipe itself and that requests for adding a new service need to be made at [NewPipeExtractor](https://github.com/TeamNewPipe/NewPipeExtractor/issues).
- [X] I have taken the time to fill in all the required details. I understand that the feature request will be dismissed otherwise.
- [X] This issue contains only one feature request.
- [X] I have read and understood the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md).

### Feature description

We want to tab for Youtube Shorts videos.

### Why do you want this feature?

for Youtube Shorts videos.

**Comment by moonstarsozluk** [comment]
Yes. It will be very good.

**Comment by mhmdanas** [comment]
Hmm, I can't exactly say I'm a big fan of the idea. YouTube Shorts don't have many differences from normal YouTube videos, so I don't see the need for a separate tab.

**Comment by alittlebitofit** [comment]
@mhmdanas It's an entirely difference experience actually.
Have you used YouTube shorts tab in YouTube app, or on mobile version of YouTube?

**Comment by mhmdanas** [comment]
I haven't used the YouTube app in a couple years, and I think YouTube Shorts are an abomination anyway, but I'll take you at your word. It's up to the team if they want to have this feature though.

**Comment by opusforlife2** [comment]
If this is ever implemented, I would really hope there is an option to turn it off. Or better, have the option off by default. No need to exacerbate the generational shortening of attention spans.

**Comment by wtusg** [comment]
Nonetheless Shorts' like player interface may be suitable for any vertical video on phones.

**Comment by opusforlife2** [comment]
> Nonetheless Shorts' like player interface may be suitable for any vertical video on phones.

@wtusg The proper solution for that is to improve the auto-full screening behaviour of the app. It's a pretty core functionality, so I don't think we should settle for any workarounds.

**Comment by B0pol** [comment]
Also there is no such thing as "trending" for youtube shorts, it’s just recommendations

**Comment by dmint789** [comment]
I fully support this as someone who hates youtube shorts. Please give them a separate tab and make it possible to just turn them off and never see them.

**Comment by mhmdanas** [comment]
@dmint789 that's a good point actually.

**Comment by bogdanmarinescu** [comment]
please add an option to disable this obnoxious type of content

**Comment by ins2d** [comment]
Indeed, something as simple as 'show/hide shorts' ( like 'show/hide watched' in the header now (or [x]_show_upcoming) ) would be incredibly useful!!!

The idea is to be able to completely get rid of Shorts anywhere in the app. But also to have a quick access when needed (so prob won't work very well if buried deep into settings).

P.S. Keep up the good work guys! <3

**Comment by arfanmrafeek** [comment]
The URL for shorts is like this **https://m.youtube.com/shorts/ ...........**
Using this, Actually you can implement Shorts tab easy and fast. Hope for it...

**Comment by seniorm0ment** [comment]
I would like to see this as well, I do agree I fail to see the issue with simply adding it as a tab that users can enable or disable just like all the other tabs.

I would hope it can tailor content to the user similar to how it works on the website. Even if not logged in, it seems to recognize how long you remain on certain videos and keep feeding you that content, I presume if not logged in it uses cookies. So for example if a video of boxing comes on, I watch it fully, then next to a non boxing video and I skip it, then a boxing video comes back on and so on it recognizes I like boxing videos.

**Comment by EvilGremlin** [comment]
+1, option to put all shorts to separate tab, but more than that option to simply hide them is sorely needed. Well, maybe show shorts on per channel basis. But in my feed literally every single short is trash i don't want to see.

**Comment by magicweirdoo** [comment]
And I would also appreciate if there is a way to completely hide shorts in the settings.

**Comment by kyrios123** [comment]
I would also like to have a way to filter out shorts from feed

**Comment by CubeOfCheese** [comment]
I would like a YouTube shorts tab specifically for shorts from channels I'm subscribed too. Currently I'm missing those videos and don't even know about them until I look at the official YouTube app.

**Comment by CubeOfCheese** [comment]
@opusforlife2 I have it turned on. I can see the shorts when I go to a specific channel, but I don't usually go to individual channels. I usually watch videos from my subscription feed and the shorts don't show up in my subscription feed, nor would I want them to. I want to see them in a separate tab

**Comment by CubeOfCheese** [comment]
No I do not
